app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -199,6 +200,10 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
 
         setupLeakCanary()
         startMetricsIfEnabled()
+        ServiceWorkerSupport.install(
```

Comment:
We should block this on https://bugzilla.mozilla.org/show_bug.cgi?id=1759588 otherwise we'll see a crash because we're passing a closed `GeckoSession`.

Reply:
Looks like we should be unblocked here with the next A-C upgrade! 🎉